lib/node/runner/node_runner.go

```diff
@@ -579,21 +579,12 @@ func (nr *NodeRunner) waitForConnectingEnoughNodes() {
 }
 
 func (nr *NodeRunner) startStateManager() {
-	nr.RLock()
-	defer nr.RUnlock()
-
-	// check whether current running rounds exist
-	if len(nr.consensus.RunningRounds) > 0 {
-		return
-	}
-
 	nr.isaacStateManager.Start()
 	nr.isaacStateManager.NextHeight()
 	return
```

Comment:
That return is also unnecessary

Reply:
> That return is also unnecessary

@Geod24 Yes right. It is just coding style.

Reply:
Unnecessary, but it is no harm.